portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,30 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function addRandomQuote() {
+  const quoteContainer = document.getElementById('quote');
+  const authorContainer = document.getElementById('author')
+  fetch("https://type.fit/api/quotes")
+    .then(response => {
+      return response.json();
+    })
+    .then(data => {
+      const quoteAndAuthor = data[Math.floor(Math.random() * data.length)];
+      const author = quoteAndAuthor.author;
+      const quote = quoteAndAuthor.text;
+      // Some quotes do not have author and the API return none.
+      if (!author) author = '';
+      quoteContainer.innerText = quote;
+      authorContainer.innerText = author;
```

Comment:
Instead of `quoteAndAuthor` this object should have a type-like name to indicate its purpose as an entity rather than the sum of its parts. `quote` works well here, I think.

It isn't necessary to extract the individual fields from this object before using them.

All control blocks require braces `{}`. See https://google.github.io/styleguide/jsguide.html#formatting-braces-all

`author` is reassigned in the `if` statement but it is declared `const`. This will throw an error if this statement is reached.

My suggestion is the following:

```javascript
const quote = getRandomElement(data)
quoteContainer.innerText = quote.text;
authorContainer.innerText = quote.author || '';
```
and `getRandomElement` defined somewhere in the file

```javascript

/** 
 * Fetches a random element from an array.
 * 
 * @param {TYPE[]} array the source array
 * @returns TYPE
 * @template TYPE
 */
function getRandomElement(array) {
  return array[Math.floor(Math.random() * array.length)];
}
```
See https://google.github.io/styleguide/jsguide.html#features-functions-generics

Reply:
I totally missed the const part. Thanks !
Changes made.